Automattic/wp-calypso
# WP Job Manager Extension: update data layer handlers to dispatchRequestEx

Another step in #25121.

#### Testing instructions

Disable `calypsoify/plugins` and install WP Job Manager to your Atomic/Jetpack site. Go to the plugin settings page. Check that everything works there (setup, pages, job listings...)

I personally didn't manage to make the plugin work. Even after installing the plugin, when Calypso is sending requests to the "Jetpack proxy" endpoint:
- URL: `/rest/v1.1/jetpack-blogs/139083458/rest-api/`
- `path` param:  `/wpjm/v1/settings`

I always get a 404 (enveloped) with error: `rest_no_route`.

Maybe someone else knows how to fix that? Maybe @donnapep? Was the extension even ever finished?
